pennylane/vqe/vqe.py

```diff
@@ -221,7 +250,7 @@ def _obs_data(self):
             tensor = []
             for ob in obs:
                 parameters = tuple(
-                    param.tostring() for param in ob.parameters
+                    param.tobytes() for param in ob.parameters
```

Comment:
@Lucaman99 I thought you were going to revert this?

Reply:
OH, right! Will do